### ISSUE TITLE
Metamorphoses 2.0

### ISSUE THREAD

**Issue by teagan75** [teagan75/Metamorphoses#35]
As I am almost finished with Metamorphoses 1.10 version, I was thinking about porting it to 1,12. This will allow the pack to be on servers and will hopefully solve the issue of mods disappearing from Curse.

Harvest Festival was ported to 1.12 and I am waiting for the Harvestcraft compat to be released. There are quite a few magic and tech mods that didn't update to 1.12 so I have found replacements that are easy to configure to be a part of the progression and help. I also added TekTopia so I can actually have players make dimension outposts. Even minecolonies can be configured to allow multiple towns and ones in other dimensions. Below is the new list of mods:

Immersive Energy (by CrimsonDragonRider)
Better Builder's Wands (by Portablejim)
Tree Growing Simulator (by tterrag1098)
Obsidian Ore (by Lordepos)
Vegan Life (by rademax93)
Actually Additions (by Ellpeck)
AgriCraft (by InfinityRaider)
Ender Hopper (by Lemonszz)
More Overlays (by FeldiM245)
Grass Seeds (by Serilum)
Redstone Flux (by TeamCoFH)
Tektopia Harvestcraft Compat (by almightytallestred)
Harvest Festival Legacy (by Snownee_)
Dynamic Trees - Pam's Harvestcraft Compat (by ferreusveritas)
Dirt2Path [FORGE] (by mallrat208)
Dynamic Trees Roots (by Dumdidldum)
Immersive Tech (by FerroO2000)
Avaritia 1.1x (by brandon3055)
Immersive Intelligence (by Pabilo8)
nether+ (by bmaster_gamer)
The One Probe (by McJty)
JAOPCA (by TheLMiffy1111)
Custom NPCs (by Noppes_)
IE Cloche Compat (by senilepenguin)
Pam's HarvestCraft (by pamharvestcraft)
NetherEx (by LogicTechCorp)
FTB Backups (by LatvianModder)
Netherending Ores (by ic_trab)
Carry On (by Tschipp)
Harvest Festival Crop Reader (by Ennek)
Refined Storage (by raoulvdberge)
[SBM] Wooden Shears (by BuiltBroken)
Random Bone Meal Flowers (by Serilum)
Wither Skeleton Tweaks (by Shadows_of_Fire)
Immersive Engineering (by BluSunrize)
Barrels, Drums, Storage & More (by Funwayguy)
NetherBound (by theishiopian)
Nature's Aura (by Ellpeck) ? not sure i am keeping this in
CraftingCraft (by BlayTheNinth)
Custom Starter Gear (by brandon3055)
Runes of Wizardry (by xXilef11)
Blood Magic (by WayofTime)
Pam's BrewCraft (by pamharvestcraft)
Immersive Petroleum (by Flaxbeard)
More Furnaces (by cubex2)
Iron Backpacks (by gr8pefish)
Prefab (by wuestman)
Hopper Ducts (by FyberOptic)
FTB Utilities (by LatvianModder)
Simple Cobblestone Generator (by LuHeGi)
Fast Leaf Decay (by olafskiii)
ezWastelands (by ezterry)
Speedy Hoppers (by al132)
Just Enough Items (JEI) (by mezz)
Baubles (by Azanor13)
Just Another Rotten Flesh to Leather Mod (by alexdaum1)
RiteClicker Mod (by OCDiary)
Ender-Rift (by gigaherz)
MineColonies (by H3lay)
Bad Mobs (by DarkhaxDev)
FTB Library (by LatvianModder)
Varied Commodities (by Noppes_)
Canny Composter (by Stormwind99)
Unique Crops (by BlueNeenja)
Better Questing (by Funwayguy)
Better Questing - Standard Expansion (by Funwayguy)
BetterBedrockGen (by GigaBit101)
Craftomancy (by CactusCoffee)
Immersive Cables (by SanAndreasP)
Bedrock Ores (by Sangar_)
Akashic Tome (by Vazkii)
Refined Storage Addons (by raoulvdberge)
End: Reborn (by elecatron)
Edible Bugs (by Eracyhw)
Dynamic Trees (by ferreusveritas)
TekTopia (by TangoTek)
Foamflower (by kamildanak)
Tumbleweed (by konwboj)
Sparks Hammers (by bright_spark)
Default World Generator (port) (by ezterry)
Tiny Mob Farm (by davidmaamoaix)
JourneyMap (by techbrew)
Lore Expansion (by theCyanideX)
Schematics (by Dom_Codes)
Astral Sorcery (by HellFirePvP)
Mystical World (by EpicSquid319)
Chest Hoppers (by SimplySimon0)
Roots (by EpicSquid319)
Runes of Wizardry - Classic Dusts Pack (by xXilef11)
Arcane Archives (by AranaiRa)
Refined Avaritia (by PricelessKoala)

Would love any feedback and/or suggestions :)

**Comment by Ladysarajane** [comment]
Love the ideal of Tektopia. I've played with that and totally love it. I've don't remember playing with Serene Seasons or AgriSeasons but they seem to mimic Harvest Festival seasons. Maybe some basic Schematics to start people with TekTopia. I know to get people started they do need some iron and I think a diamond to make the Town hall token so they can get other tokens in a chest. .  Not sure if a given or awarded token will work to spawn the chest. 
 Immersive engineering starts basic. Progressive automation I had trouble with at times, but it has been awhile. Hopefully it has improved. 
Festivals for me in Metamorphosis didn't always work. The Harvest festival with the communal pot...the people showed up... but that was it. Chicken festival never happened other than the people talking about it and a letter showing up. Only ones that worked were the sheep, cow and New Years Eve. So I won't miss them. My animals even though they won, never did give better products like promised. So I am assuming the Prefab buildings are not for version 1.12 of minecraft? I Liked the tree farm, farm and greenhouse. The fields were great too, if you are using a barren world. 

Oh...add the mod Bed Bugs. I have gotten stuck in bed and Bed bugs help eliminate some of the "unable to get up" bug.

**Comment by teagan75** [comment]
HA Bed Bugs the bug Mojang never fixes!
Prefab will be in the pack along with Minecolonies so TekTopia was going to come later for outposts (if they work lol) I mainly noted the new mods or changes. Everything else ported to 1.12

Oh and I was thinking of copying the HF buildings and bringing them to 1.12 using the mod Schematic that pops in the buildings. Also thought about adding in Twilight Forest but not sure

**Comment by Ladysarajane** [comment]
Keeping Prefab yes!!! I don't know if Minecolonies and TekTopia will play nice together. Never seen it done. It would be nice though. Twilight forest I am not to fond of. I am not really much of an adventurer. I am more of the minecolonies and Tektopia type of person. Barbarians and Necromancers are enough bad guys for me...LOL. Schematic mod sounds interesting.

**Comment by teagan75** [comment]
I used to like Twilight Forest a lot until they made it where you had to fight the bosses in order to open more of the world. I was just thinking it would be nice as a lumbering outpost because of the large trees.

My thoughts are to still use minecolonies in the overworld and see if tektopia will work in other dimensions. Like the miner could mine in the nether or the mining dimension. I could still use twilight forest but use Mystcraft to make a world. Then no bosses hehe. I am still playing around with the mods at this point to see what works and what would just be silly

**Comment by teagan75** [comment]
this mod may actually give the same feel as HF mining dimension. You progress through the caves as you grow stronger as a miner. Special mobs show up too with special items. I will have to play around with it lol. The mod list for the port to 1.12 is still very WIP. Certain things are set others aren't hehe. If tektopia works in other dimensions (which minecolonies doesn't) then i will remove Progressive Automation because you won't need it. Things like that

https://www.curseforge.com/minecraft/mc-mods/cavern2

**Comment by Ladysarajane** [comment]
Cavern2 looks interesting. I don't think TekTopia works in other dimensions. If I remember correctly some people tried it during beta and it wouldn't work. I think it was a mining dimension they tried. I don't think TangoTek is supporting the mod any more either. Seems to have abandoned it. Issue section on github is no longer there. No activity since last October from him. :(

What about simple storage Network mod for a basic storage system?

**Comment by teagan75** [comment]
I was going to try this Barrels, Drums, Storage & More

**Comment by teagan75** [comment]
Tektopia is working in the cave dimension 
![2020-05-20_11 06 44](https://user-images.githubusercontent.com/12485764/82469748-0fd4f380-9a8a-11ea-99b4-c7b4e6cf5ae3.png)

![2020-05-20_11 06 46](https://user-images.githubusercontent.com/12485764/82469763-1499a780-9a8a-11ea-93b4-7dee01934f43.png)

**Comment by Ladysarajane** [comment]
Barrels, drums and storage and more is good and GREAT love that TekTopia works in the Cave dimension. As I said, I think it was the Aroma mining dimension that it didn't work.

**Comment by teagan75** [comment]
once I determine which mods I want to keep I will post the whole list here

**Comment by teagan75** [comment]
OMG I just saw someone ported Harvest Festival LOL

**Comment by Ladysarajane** [comment]
Ported it? Whoa! Need to look that up. That will help alot.

**Comment by teagan75** [comment]
Yep. Ported to 1.12! So this whole thread is pointless now lol. Except for discussing magic and tech mods. Tektopia will stay. 

When I looked months ago it wasn’t there.

**Comment by Ladysarajane** [comment]
Yes..I think it was early Jan, when I last look also. 

Edit: Anxious to try out the new beta version of the pack you are making now...LOL

**Comment by teagan75** [comment]
LOL me too! Also, the dev that did the Pam's Harvestcraft compat is working on updating his mod too. Woot!

**Comment by Ladysarajane** [comment]
That's great! Pam's working with Harvestcraft and Minecolonies 1.12 lumberjack will be easier to control. I had a problem with him and that park (even though it should have been out of his reach) and my lumberjacks kept climbing the huge trees in the Tree farm and getting struck and killed by barbarian archers.  Getting more excited!

**Comment by teagan75** [comment]
I had the same issue with the lumberjack. I haven't played much with Minecolonies 1.12 and I see a lot of new buildings. I hope they work. 

So, I am cleaning up the quest book because some mod IDs changed. Also, I am redoing the Roots section. After that I may release a beta version with some quest lines WIP. I will soon update the OP with the mod list

**Comment by Ladysarajane** [comment]
The forge building in the version of 1.12 was in development and did not quite work...although a few updates came out after that pack was developed. (The Simply Life 2 Genesis by 99blocks).  Building was good though. Request system was better and event update handling improved not causing as many crashes. The resource scroll worked better, but would occasionally crash the client. A library was added to increase villager stats, a composter (if you use that, make sure it can compost bugs please.) Bugs become annoying after you have other foods.) Lumberjack was improved and you could place cobble under full grown trees to keep him from chopping them. The stone mason was iffy in early 1.12....sometimes he did his job and other times he didn't. The mailbox request system would work for me if I didn't overwhelm it. ( you could request stairs or other items and the deliveryman would put it in the box and you could give it to the people that needed it. ) It would be good to play with Minecolonies 1.12 again. 
Looking forward to trying the beta version.

**Comment by teagan75** [comment]
I assume the composter is for farmland. Not sure if composted dirt will affect HF crops. Not much does.

**Comment by Ladysarajane** [comment]
The composter is also for the florist shop...I think it was added in 1.12. The combat academy, Stonemason,  Forge, library (for leveling citizens up}, Archery {levels up archers), bakery, florist, blacksmith,  I believe became active in 1.12.2. You can have the blacksmith make armor and weapons for the Colonies and you. Also other items made in metals.  You teach them recipes to make related to their job. (ex: builder can make slabs or torches)  A postbox also available....although I found it extremely laggy to load up available items to make. Not sure if Raycomms found a solution for that in 12.2.  I don't think the Enchanter became available until 1.15

**Comment by teagan75** [comment]
I love all the additions. Makes up for not having Village Box

**Comment by Ladysarajane** [comment]
They are awesome. I have watching the Minecolonies Developer streams and the new 1.16 stuff is awesome...the Villagers use minecarts and they have incorporated the tracks into some of the buildings. Makes deliveryman faster. The new schematics also very neat in 1.16. Asian style and medieval buildings are awesome. Enchanter towers....pretty cool when 1.16 minecolonies is released. 

Edit: I forgot in 1.12.2, Children are a thing. Not sure if the school was added in 12.2 or in 1.15..but the alpha minecolonies has a school and to unlock certain buildings you need a University to do research. Pretty cool.

**Comment by teagan75** [comment]
Wow! Sounds awesome. Just need more mods to port over. Prefab has a couple new buildings too. I am thinking of adding a quest for the saloon. I added Pam's Brewcraft to the pack

**Comment by Ladysarajane** [comment]
Great really anxious to try this.....Didn't know that the saloon was in prefab.

**Comment by teagan75** [comment]
minecolonies has become very customizable. I love that as a pack dev. For instance the sifter. Most of that stuff would be only worth putting into the biofuel reactor. I mean that may be worth it to some people but I need to come up with a better list for the pack. I have a lot of config work to do 

Edit: Minecolonies allows more than one town now and ones in other dimensions! So the miner could work there. Not sure if its's buggy or not which is why I have TekTopia hehe

**Comment by Ladysarajane** [comment]
Neat.....I know you could be an officer in another town...but I didn't know you could own more than one town now. I know they were working on other dimensions....didn't know if it was there yet. My save became corrupt when I was playing and I never restarted it and they were adding stuff then. Anxious to get back into it again.

**Comment by teagan75** [comment]
Since you have played minecolonies, let me know if I should rearrange when certain buildings were built in the quest book. I am trying the new stuff out now but welcome any suggestions

**Comment by Ladysarajane** [comment]
Sure....I would be glad to. In 1.12 colonists really depend on the restaurant for eating, so that is important early. They will go there automatically to get food. That should be kinda early in a quest line.

**Comment by teagan75** [comment]
I was thinking about that too. Also, depending on how many citizens I have in testing I may add more housing in later chapters. Here is what I have:

Settlement:
Town hall
builder
guard tower
houses

Hamlet:
lumberjack
sawmill
Delivery
Warehouse

Village:
more houses
more guard towers
restaurant
crusher
stone smeltery

City:
more houses
stonemason
ore smeltery
composter (bugs have been added to list)
florist (configured to place some mod flowers too)
barracks

Metropolis:
Sifter (configured to use wasteland dirt)
Blacksmith
library
enchanter
bakery

Capital:
TekTopia Nether Outpost (by this time you should have tesla coil defense and maybe other advanced weaponry)

Empire:
TekTopia End Outpost

**Comment by Ladysarajane** [comment]
I think I would put the barracks earlier under city. Guards then would not need houses in the barracks. Citizens happiness would be increased. A good Colony needs 4 guard towers and the rest barracks. The guards in towers require a house...the barrack guards sleep in the barracks. Towers in the barracks are built one at time, then they can be leveled up. Barbarians are a little more organized in this version...they climb, break blocks etc and you do need the barbarians if you want to level up your town hall to level 5. The last I saw the town hall required the barbarian Chief sword to level up to level 5.  A warehouse and deliveryman would be important around the Village stage so a player wouldn't have to keep supplying the builder or restaurant with raw materials. I am not sure if Raycomms was able to get the farmer to plant and harvest sugar cane...but that would be good for the restaurant and later the bakery, if the farmer could. It would be the only use for a farmer if he did get it.

**Comment by teagan75** [comment]
I can definitely shift things around. Then in the later two chapters focus on tektopia in the other dimensions

**Comment by teagan75** [comment]
Ok updated the list and the quest book

**Comment by Ladysarajane** [comment]
Looking good. Can't wait to try it to see how it goes. I should play the latest version of 1.12 Minecolonies to see if I am missing something we may need. It has been awhile since I played.

**Comment by teagan75** [comment]
I am very close to an early beta release. I am nearly done cleaning up the questbook and making sure resource progression is attainable since Alchemic Ash is not in the pack

**Comment by teagan75** [comment]
Original post updated. I will be releasing a rough beta version. Nothing should crash hehe. The first few chapters I believe are complete mostly. I am not done replacing out some of the quests and writing narrative in new quests. It's definitely rough

**Comment by Ladysarajane** [comment]
List looks good. I will be waiting for the beta version.

**Comment by teagan75** [comment]
It’s been approved. And quickly too!

**Comment by Ladysarajane** [comment]
Great...Is it called Metamorphosis 2?

**Comment by teagan75** [comment]
No I didn't make a separate project for it. Would have taken a long time to get approved.
You can update to a new profile. I may consider making a new project after the holiday weekend.

**Comment by Ladysarajane** [comment]
Okay...it might take a couple hours for it to show. I will make a new profile for it.

**Comment by Ladysarajane** [comment]
*smacks head* It dawned on me that it won't show up as an "upgrade". Had to go to settings and versions and download it from there. I will have to try it tomorrow...getting late and storms in area.

**Comment by teagan75** [comment]
No problem. Maybe by tomorrow my project page will be approved hehe. Stay safe. We are getting storms tonight too

**Comment by teagan75** [comment]
Btw, if you downloaded the file last night i think you can still play it. I had to delete the release from the old project so I could use it for the new project page which hasn't been approved yet.....

**Comment by Ladysarajane** [comment]
I'm playing it now...so far it has been good. Astral Sorcery and and End mod structures
![2020-05-25_08 32 07](https://user-images.githubusercontent.com/34258851/82818119-3c16b880-9e6c-11ea-90b6-061f8cbf57b4.png)
![2020-05-25_08 30 10](https://user-images.githubusercontent.com/34258851/82818135-41740300-9e6c-11ea-91d3-4f5db79373c3.png)

 have been spawning structures...tumbleweeds are soooo cute. Biomes seem better too. I'm not in a deep ocean or desert biome all the time now. Composting is good...but that recipe for Grassy plains is a pain at the beginning. I know it won't be a problem later....but I'm addicted to those plains...LOL. Silkworms....I forgot how those work. Been playing in 1.7 too long. I think I spawn them and collect the silk they drop. 

If you did want Astral Sorcery structures...I was afraid the temples wouldn't spawn on the lower levels but they are fine...
![2020-05-25_08 41 39](https://user-images.githubusercontent.com/34258851/82818239-69636680-9e6c-11ea-8252-5cbe93098416.png)

**Comment by teagan75** [comment]
RE: end structures; a bit of the end seeped through during the Great Ender War 😄 

RE: astral sorcery; I am glad they spawn. Tiberius also sells a collector crystal and you can transmute stone brick into marble via the atomic reconstructor. The means to get rock crystal and aquamarine is through Arcane Archives.

**Comment by Ladysarajane** [comment]
LOL that sounds good. I haven't dug down yet to see if the structures are intact...but I heard mobs in daylight under them...so I am assuming. So far everything is running well. Can a person turn off those tumbleweeds if they have a slower computer? Not a problem for me but I could see it might be a problem for some. They don't spawn too often.  I am being slow and I see mortar and pestle can break down the silk worm droppings. Also a spindle...but don't have iron yet to make a tripwire. 
Liking this so far in 12.2. 
Not sure if it's Pam's or I am being unlucky...but I haven't gotten one apple off the trees yet.

**Comment by teagan75** [comment]
you know, I noticed the apple thing too but thought I was being unlucky. It may actually be a Harvest Festival thing. I will need to look in the configs for both mods. I noticed HF has their own apple tree. Pam's never messed with vanilla apples before but that could have changed.

String can still be made with the drops from jungle trees and you get those saplings in the second chapter like before

**Comment by teagan75** [comment]
Also, I didn't realize I needed COfh World to spawn ores in the nether and end.....whoops! Will have to add that next update hehe

**Comment by Ladysarajane** [comment]
I'm glad you noticed the apple thing too. I thought I was being unlucky. Love that the bugs can be composted. The HF might be the culprit if they have fruit trees now. For string I get spiders too. Just love playing with the silkworms. Mine is pooping silk regularly now...LOL. Can't wait till I get to tags and a anvil to name him.   Oh, glad that extra silkworm eggs are compost able. They tend to add up quickly.  Expecting heavy storms again this afternoon....just in case I disappear. (I'm storm spotter and Amateur radio operator.  Where did Spring go? In the forties Tuesday and now in the 90's. Good thing we had already started setting the pool back up (metal intex pool). Still a little chilly but feels good until the storms come. Stupid storms already ripped the brand new replacement cover, first day we put it on. Storms are not organized just the summer popup variety, but they last awhile and dump a lot of rain on areas that are already saturated. (Our pool got filled up with it...one good thing...we get to have a lower water bill LOL.

**Comment by Ladysarajane** [comment]
Okay...since you are starting another Project page I didn't want to create an issue here but I found first bug. Minecolonies. Cannot craft the town hall. 
![2020-05-25_11 47 34](https://user-images.githubusercontent.com/34258851/82827855-d501ff80-9e7d-11ea-9ba8-547acb82f026.png)
![2020-05-25_11 47 41](https://user-images.githubusercontent.com/34258851/82827861-d8958680-9e7d-11ea-8163-ca0998be8e05.png)

Is this a custom recipe? I thought it was suppose to be a door with wood around it with the builders wand?

**Comment by Ladysarajane** [comment]
Now a weird glitch. I went to bonemeal grass and it worked...but a grass block disappeared. 
![2020-05-25_11 52 06](https://user-images.githubusercontent.com/34258851/82828093-578abf00-9e7e-11ea-9591-e8ee08496931.png)

Edit...HA HA...when I went to replace it with dirt...it worked but it immediately turned into grass block.

So Yuliff isn't suppose to sell beds anymore?

**Comment by teagan75** [comment]
Ok lots of differences lol. I didn't look to see if Yulif sold beds. I assumed he did. I will add it. Weird glitch with bonemeal. Wonder what that could be? I assumed the town hall was craftable because the recipe showed up. I will have to add one

**Comment by Ladysarajane** [comment]
Okay on the townhall, is it looking for an anvil too before I can craft it?
![2020-05-25_12 13 24](https://user-images.githubusercontent.com/34258851/82829506-6fb00d80-9e81-11ea-9e7d-6424685882f0.png)

**Comment by teagan75** [comment]
no that has a red X meaning you cannot craft it in an anvil
not sure why the furnace is green tho lol

**Comment by Ladysarajane** [comment]
I crafted a furnace already. I assume it was looking for those items with the green on the crafting table and furnace since I had crafted them...LOL

**Comment by teagan75** [comment]
ok so instead of two copper coins for completing the first chapter I am gifting 2 HC plum saplings. We need to feed our workers and the apple tree from HF is only plantable in Autumn. So, to not break the system I am using a fruit tree that doesn't have a HF counterpart.

**Comment by Ladysarajane** [comment]
Sounds good.

**Comment by teagan75** [comment]
Just a heads up I am going to disable structure generation. I want to gate Astral sorcery and the roots huts don't fit the theme lol

**Comment by Ladysarajane** [comment]
Okay......I haven't seen the roots huts yet.

**Comment by teagan75** [comment]
on second thought I may just disable the large shrine and keep the small ones that don't have the collector crystal. This way marble will be easier to get. The roots huts aren't a big deal

**Comment by Ladysarajane** [comment]
Okay....they do have loots in them(astral sorcery) . I just checked. Aquamarine, iron, glowstone and the star papers so far.

**Comment by Ladysarajane** [comment]
In regards to the Minecolonies townhall....I gave myself a prefab field to have a level grassy area (had to add dirt blocks for the 17x17....then I crafted a camp. I did find a townhall in the chest in the camp, but I attempted then to craft the townhall and I was able to. So somewhere there must be a setting that checks to see if a camp or supply ship is placed before it allows crafting.

**Comment by teagan75** [comment]
It’s ok. Every time you pop in the HF buildings they come with loot

**Comment by Ladysarajane** [comment]
Another crafting problem. Wooden crates share the same crafting recipe as chests.

**Comment by teagan75** [comment]
ah yes forgot to change that

**Comment by teagan75** [comment]
Also, for grassy plains you can use wasteland dirt for compressed dirt. It's just the grass then that is the issue. Also, a minecraft seed used on dirt will turn it to grass

**Comment by Ladysarajane** [comment]
Oh...I didn't know that. That's wonderful! I have the grass clippings.
They've activated storm spotters in our area....just had line develop over
us, but it moved east. Kids (my niece and nephews) in the pool now before
next line comes in about an hour. Pool too cold for old me.

**Comment by teagan75** [comment]
It is a hot day today. We have a small storm headed toward us now.

**Comment by teagan75** [comment]
I love that you can see the boundaries of your town when holding the build scepter

**Comment by Ladysarajane** [comment]
Yeah you could in 1.7 but you had to placing a building to see it. If I'm not mistaken I think boundaries are now flexible...if a building is placed inside, but close to a border, the border expands on that side. Don't know if they kept that in 1.12 release, but it was in the beta. I stopped playing when my town loaded...but my people all froze and we couldn't figure out why. My town was big then. Reverting to previous save didn't work then...so I quit playing it. Went on to other packs.

It was 88 here and after the first line passed, it is down to 72. Hopefully that will stabilize the atmosphere and keep the bigger storms from having power.

**Comment by teagan75** [comment]
Your question about tumbleweeds: there is no easy way to turn them off but only 8 can be in an area around a player and since most mobs are turned off it shouldn't be a big deal

**Comment by Ladysarajane** [comment]
> Your question about tumbleweeds: there is no easy way to turn them off but only 8 can be in an area around a player and since most mobs are turned off it shouldn't be a big deal

I've noticed that the same two tumbleweeds keep repeating about 32 blocks away. Adds a nice ambience to the pack.

**Comment by Ladysarajane** [comment]
Just realized that in beta we did not start with a backpack but in the Version 1.0 a backpack was starting equipment. Are we not starting with backpacks this time?

**Comment by teagan75** [comment]
> Yeah you could in 1.7 but you had to placing a building to see it. If I'm not mistaken I think boundaries are now flexible...if a building is placed inside, but close to a border, the border expands on that side. Don't know if they kept that in 1.12 release, but it was in the beta. I stopped playing when my town loaded...but my people all froze and we couldn't figure out why. My town was big then. Reverting to previous save didn't work then...so I quit playing it. Went on to other packs.
> 
> It was 88 here and after the first line passed, it is down to 72. Hopefully that will stabilize the atmosphere and keep the bigger storms from having power.

There is a config option to turn dynamic city growth on. I assume this is what you are talking about. I can change it to true. For the server package I will keep it false cause I think that might cause issues.



> Just realized that in beta we did not start with a backpack but in the Version 1.0 a backpack was starting equipment. Are we not starting with backpacks this time?

You are right. I will add that to the list of starting gear. It should have been there since I just moved the config file over from 1.10

Edit: the item ID changed lol

**Comment by Ladysarajane** [comment]
Yeah...sounds good, dynamic for single player and not for server. I like dynamic cause I always had a small sliver of a building that  needed that small piece to place.

Thanks for fixing the backpack. I'll wait to next update to play again. Stuck at the storage barrels and apple sapling quests.

**Comment by teagan75** [comment]
Curse is taking its sweet time approving my new project :(

**Comment by Ladysarajane** [comment]
Yeah....they will blame Memorial Day.

**Comment by teagan75** [comment]
Yes they will. So, Minecolonies has changed a bit. Do guards not need housing? I have four houses and cannot assign my two guards to the houses. Do I need to build my town hall now and have housing to get more citizens? I wish their wiki was up :(

Edit: Wait, do I have to recruit now? They just don't come?

**Comment by Ladysarajane** [comment]
Yeah...guard towers and barracks house the guards now. Actually level your builder up to level 2 and upgrade some of the houses to level 2. A child should spawn. I don't know if you can entice a citizen at level 1 in town hall (If there is room in a house, either a child will spawn if a male and female are assigned to the same house and there is an extra bed in town, but you can "bribe" a citizen to come...there is a page in the town hall book for that, but I cannot remember what level of town hall it becomes available.

As to the wiki, he said in a developement stream over the weekend that the wiki server crashed and they are waiting on new equipment.

**Comment by teagan75** [comment]
Yeah I did recruit one and my town hall isn't built but that will get expensive. I will have to make a note in the book that the builder and citizen huts need to be level two so a child can spawn. WOW, progression will be much slower lol

**Comment by Ladysarajane** [comment]
Yeah...i try to get a lumberjack asap. Cobble and glass are the next. For building.. Resource scrolls are better in this version, but occasionally it does crash the client. If a citizen (esp the builder gets stuck make sure they have their requested item, if the do and still don't work...fire them wait till clothes change to regular clothes and rehire them. They retain stats. One reason I turn off automatic hiring. Once you get warehouse and delivery man, you can make stairs etc in bulk, put them in the warehouse  and the delivery man will take them as needed to builder

**Comment by teagan75** [comment]
the person i hired for sawmill isn't working. I added recipes too. I fire and rehire a lot hehe. I put what they want in first inventory slot and it seems to fix em.

**Comment by Ladysarajane** [comment]
Yeah...usually it is the builder that gets hung up. Also if they want something I give it to them directly and cancel their request on them.

**Comment by teagan75** [comment]
Yay! Wiki is back up.

I was thinking now that we have a crusher and stone smelters, I was going to try and hook the cobble gen up to a chest in the warehouse

**Comment by Ladysarajane** [comment]
Yay! Equipment hooked up! I was going to try that( the cobble gen in the warehouse,  but when you upgrade the warehouse, you'll need to remove the cobblegen to keep the builder from destroying it.

**Comment by teagan75** [comment]
yep! Can't be forgetful lol

**Comment by teagan75** [comment]
Ok. So the new project page has been approved and I put out another release better than the first I hope lol. I am going to close this out because there is a new issue page
